source/index.html.md

```diff
@@ -200,6 +200,9 @@ error categories, the last two digits define specific errors.
 * 0366: Two factor authentication method is not enabled
 * 0367: Unable to generate quote at the present time
 * 0368: Rejected withdrawal to restricted address
+* 0369: File type not allowed
```

Comment:
Seems this error code is already taken [here](https://github.com/bitsoex/slate/pull/83)